example/src/main/java/io/netty/example/http/websocketx/client/WebSocketClientHandler.java

```diff
@@ -82,9 +83,14 @@ public void channelInactive(ChannelHandlerContext ctx) {
     public void channelRead0(ChannelHandlerContext ctx, Object msg) throws Exception {
         Channel ch = ctx.channel();
         if (!handshaker.isHandshakeComplete()) {
-            handshaker.finishHandshake(ch, (FullHttpResponse) msg);
-            System.out.println("WebSocket Client connected!");
-            handshakeFuture.setSuccess();
+            try {
+                handshaker.finishHandshake(ch, (FullHttpResponse) msg);
+                System.out.println("WebSocket Client connected!");
```

Comment:
@normanmaurer Is it necessary to replace it? The code is an example. It's easier to grab code with println and run it somewhere without configuring of a logger..